### DISCUSSION TITLE
Is it possible to use Just in Time features in Twin Macro? (custom input values)

### DISCUSSION THREAD

**Discussion by dredshep** [ben-rogerson/twin.macro#598] [Q&A] [answered]
Hello, I'm thinking of integrating `twin.macro` into a new project but I would like to have Just-In-Time for it, stuff like `text-[#123456]`. Will I have this? Is there a way to enable it or to use Tailwind v3 with twin macro?

**Comment by ben-rogerson** [comment, marked as answer]
Yes, that will work in the current version of Twin.
Here's a list of [the other arbitrary values](https://github.com/ben-rogerson/twin.macro/blob/master/__fixtures__/arbitraryValues/arbitraryValues.js) you can use.

Check out [the JIT release notes](https://github.com/ben-rogerson/twin.macro/releases/tag/2.7.0) for more info.
